src/test/app/claims/ccjModelConverter.ts

```diff
@@ -0,0 +1,141 @@
+import { expect } from 'chai'
+import { CCJModelConverter } from 'claims/ccjModelConverter'
+// Claim Data
+import { fullAdmissionWithImmediatePaymentData,
+  fullAdmissionWithPaymentBySetDateData,
+  fullAdmissionWithPaymentByInstalmentsData,
+  partialAdmissionWithImmediatePaymentCompanyData,
+  partialAdmissionWithPaymentBySetDateCompanyData,
+  partialAdmissionWithPaymentByInstalmentsCompanyData
+ } from 'test/data/entity/responseData'
+// Claimant-Response Draft and Data
```

Comment:
please don't add comment in imports, IDEs will move them around when they are refactored and comments won't be updated

Reply:
no problem, I will refrain from doing so. I only put them in to help me keep a track of the tasks I needed to do